docs/template-windows.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-vm-template
```

Comment:
I'd also make this a little clearer windows-2012r2-vm-template I guess.

Reply:
or maybe windows-2016? I'm fine with this choice, but AFAIK nothing is windows-2012 specific (meaning, it should work with every windows starting from 2012)

Reply:
Ugh - tehse windows version. klet's keep it as is :)